etl/steps/data/garden/irena/2023-12-12/renewable_power_generation_costs.meta.yml

```diff
@@ -1,46 +1,47 @@
-# TODO: This file is a duplicate of the previous step. It is not yet used in the dag and should be updated soon.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Energy
+    short_unit: $/kWh
+    unit: 2022 US$ per kilowatt-hour
+    description_short: |-
+      Measured in 2022 US$ per kilowatt-hour.
+
+  description_key_lcoe: &description_key_lcoe
+  - Levelized cost of energy (LCOE) estimates the average cost per unit of energy generated across the lifetime of a new power plant. It is measured in US$ per kilowatt-hour.
 
 dataset:
-  title: Renewable power generation costs (IRENA, 2023b)
-  description: |
-    Levelized cost of energy (LCOE) estimates the average cost per unit of energy generated across the lifetime of a new power plant. It is measured in 2021 US$ per kilowatt-hour.
+  update_period_days: 365
 
 tables:
   renewable_power_generation_costs:
+    title: Renewable power generation costs
     variables:
       bioenergy:
         title: Bioenergy levelized cost of energy
-        short_unit: $/kWh
-        unit: 2021 US$ per kilowatt-hour
+        description_key: *description_key_lcoe
```

Comment:
Generally speaking I'd also add more specific descriptions in description key for what each of the technologies means more specifically (either take it from source or could use chatGPT to pre-generate them and then double check it all makes sense)